includes/blocks/class-sensei-course-blocks.php

```diff
@@ -82,10 +82,11 @@ public function enqueue_block_assets() {
 			return;
 		}
 
-		Sensei()->assets->enqueue( 'sensei-single-course', 'blocks/single-course.css' );
+		Sensei()->assets->enqueue( 'sensei-single-course-blocks-style', 'blocks/single-course-style.css' );
+		Sensei()->assets->enqueue_style( 'sensei-shared-blocks-style' );
```

Comment:
We could add this as a dependency of `sensei-single-course-blocks-style` above it instead.

Reply:
Good idea! Changed here: https://github.com/Automattic/sensei/pull/3928/commits/d4b30f30712e147a43377069353c951d6d141aae